test/sql/chunk_utils.sql

```diff
@@ -169,13 +275,19 @@ BEGIN;
     SELECT * FROM test.show_subtables('drop_chunk_test_tstz');
 ROLLBACK;
 
+\dt "_timescaledb_internal".*
+
 \set ON_ERROR_STOP 0
+-- should error because all hypertables do not have the same time column
+SELECT show_chunks();
```

Comment:
In my mind, this should return all chunks from all hypertables since you have no explicit type constraints. Similarly, even if you did have type constraints, I still think you should return chunks from HTs that match the types, rather than erroring because not all HTs have the same time type.

Reply:
I have no objections to this, but we should discuss this.
I think would be great from usability standpoint.

Reply:
For no arguments that should definitely be the behavior.

Reply:
ok, fixed